### PR TITLE
Add mmention command and group mention support

### PR DESCRIPTION
**Pull request by jsvana**
`<prefix>mmention list|show|add|rm` to add nicks to a given mention group

`@<group_name>: message` will make the bot say `<nick1>[, <nick2>...]: ^`

### CODE REVIEW

File: seabird/modules/multimention.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                'Must at least supply group_name to "rm" command',
+            )
+            return
+
+        group_name = args[0]
+        nick = None
+        if len(args) > 1:
+            nick = args[1]
+
+        print('ASDF', group_name, nick)
```
Comment (belak) [inline seabird/modules/multimention.py]: Leftover debug statement

File: seabird/modules/multimention.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            self._cmd_show(msg, args)
+        elif cmd == 'add':
+            self._cmd_add(msg, args)
+        elif cmd == 'rm':
+            self._cmd_rm(msg, args)
+        else:
+            self.bot.reply(msg, 'Unsupported command "{}"'.format(cmd))
+
+    def irc_privmsg(self, msg):
+        # We need to call super here so cmd_karma can be called
```
Comment (belak) [inline seabird/modules/multimention.py]: Comment is out of date

**Comment by jsvana** [comment]
fixed

**Comment by belak** [comment]
:+1:

**Pull request merged**